rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs

```diff
@@ -243,6 +243,12 @@ impl SceneManager {
                     audio_source_node.bind_mut().set_dcl_enable(false);
                     audio_source_node.call("apply_audio_props".into(), &[false.to_variant()]);
                 }
+                for (_, audio_stream_node) in scene.audio_streams.iter_mut() {
+                    audio_stream_node.set_volume_db(-80.0);
```

Comment:
can use create a function named `set_mute(true);` that sets the volume to -80 instead of using in this way?